_guides/custom-css.md

```diff
@@ -0,0 +1,294 @@
+---
+layout: documentation
+title: Customize The Lounge on-the-fly using CSS
+---
+
+It is sometimes useful to tweak the interface to one's preferences. To do so without having to roll a full-fledged theme, or fork and maintain an existing theme, The Lounge offers an easy way to apply your own [CSS](https://en.wikipedia.org/wiki/Cascading_Style_Sheets) snippets.
+
+Towards the bottom of the **Settings** page, you will find a **Custom Stylesheet** field:
+
+![Custom Stylesheet textarea which can be found in the Settings page](/img/custom-stylesheet-textarea.png){:width="468px"}
+
+Simply add your custom CSS, and the modifications will be applied when clicking outside the field.
+
+Note that The Lounge evolves rather quickly, and new versions often change the existing HTML and CSS files. This causes custom CSS snippets to often require changes as new versions get released. Use your [browser's developer tools](https://developer.mozilla.org/en-US/docs/Learn/Common_questions/What_are_browser_developer_tools) when your trying to add or edit custom CSS.
+
+{: .alert.alert-warning role="alert"}
+Settings are not synced across devices at the moment, so any custom CSS you set will only apply to this specific instance of The Lounge.
```

Comment:
They are though?

Reply:
Ah, that was written before that feature I think :)

Reply:
Removed.